README.md

```diff
@@ -118,12 +118,11 @@ Follow along as I use mixins to make our grid system more semantic.
 </div>
 ```
 
-## Lab: Semantic Markup
+## Lab: Refactoring for 'view-state' vs. modals
 
-1.  Using the code I used above in your `index.scss` file refactor on your own.
-1.  Write your own mobile ready page using Bootstrap-Sass classes.
-1.  Refactor your HTML from earlier to be more semantic (copy/paste your
-previous code and refactor it using bootstrap-sass to make it semantic).
+-   In your squads, think about how to take the various modals we have in place and
+think about how you might use different 'view-states' to achieve the same
```

Comment:
I'd be happy to do either. I was just thinking that showing them how to think of different view states (hide/show templates) to get them thinking about handlebars/ember etc. If @J-Weeks has any insight, I'd welcome it! Happy to chat in person further.